go/chat/boxer.go

```diff
@@ -316,6 +318,12 @@ func (b *Boxer) UnboxMessage(ctx context.Context, boxed chat1.MessageBoxed, conv
 	default:
 	}
 
+	// if the server message doesn't have a TLFID, then it isn't available to us. This is most commonly
+	// used when restricted bots try to read messages not encrypted for them.
+	if boxed.ClientHeader.Conv.Tlfid.IsNil() {
+		return b.makeErrorMessage(ctx, boxed, errBoxerUnavailableMessage), nil
```

Comment:
will `ServerHeader` be `nil` in this case? `makeErrorMessage` assumes it isn't

Reply:
No, I can't do it like that, see https://github.com/keybase/keybase/pull/4964